packages/operator/pkg/webserver/routes/v1/training/model_training_validation_test.go

```diff
@@ -409,3 +412,36 @@ func (s *ModelTrainingValidationSuite) TestMtResourcesValidation() {
 	s.g.Expect(errorMessage).Should(ContainSubstring(
 		"validation of gpu limit is failed: quantities must match the regular expression"))
 }
+
+func (s *ModelTrainingValidationSuite) TestOutputConnection() {
+
+	// If configuration output connection is not set then user must specify it as ModelTraining parameter
+	viper.Set(train_config.OutputConnectionName, nil)
+	mt := &training.ModelTraining{
+		Spec: v1alpha1.ModelTrainingSpec{},
+	}
+	err := s.validator.ValidatesAndSetDefaults(mt)
+	s.g.Expect(err).To(HaveOccurred())
+	s.g.Expect(err.Error()).To(ContainSubstring(fmt.Sprintf(validation.EmptyValueStringError, "OutputConnection")))
+
+	// If configuration output connection is set and user has not passed output connection as training
+	// parameter then output connection value from configuration will be used as default
+	viper.Set(train_config.OutputConnectionName, testMtOutConnDefault)
```

Comment:
Reset the packaging_config.OutputConnectionName viper value to the nil in the test tear down.